doc/verilog_systemverilog.txt

```diff
@@ -448,6 +448,19 @@ Example:
     Note: The commands |:VerilogFoldingAdd| and |:VerilogFoldingRemove| are
     provided to allow an easier management of this variable.
 
+                                                      *g:verilog_quick_syntax*
+g:verilog_quick_syntax~
+Default: undefined
+
+    Whena enabled, syntax regions will not be defined with Vim's 'syntax'
+    command. This can help performance if you are using an old machine or
+    veiwing a large file such as a netlist. This can be useful if you don't
+    care about automatic indentation but still wish to have syntax coloring.
```

Comment:
When I was developing the new script I used `make profile` as well as editing interactively. When I added the syntax region stuff I just tested it interactively but the performance difference was very noticable.